frontend/src/main/java/puf/frisbee/frontend/view/GameView.java

```diff
@@ -99,4 +106,16 @@ private void handleButtonGameOverContinueClicked(ActionEvent event) {
 	private void handleButtonGameOverQuitClicked(ActionEvent event) {
 		this.viewHandler.openStartView();
 	}
+
+	@FXML
+	private void handleButtonQuitGameContinueClicked(ActionEvent event) {
+		this.viewHandler.openStartView();
+	}
+
+	@FXML
+	private void handleButtonQuitGameQuitClicked(ActionEvent event) {
+//		this.viewHandler.end();
+		this.gameViewModel.continueGame();
```

Comment:
Find ich ein bisschen wie cheating... wenn man am verlieren ist, klickt man quit game, abbrechen und dann fängt das level wieder von vorne an :D ... schöner wäre, wenn das game einfach an der aktuellen Stelle weiterläuft. Kann aber evtl. auch in ein extra Ticket?

Reply:
Stimmt, ist aber gerade nur der Countdown, der dann noch einmal von vorne beginnt. Kann ich vielleicht in diesem PR noch lösen.